assets/tests/factories/shape.ts

```diff
@@ -0,0 +1,8 @@
+import { Factory } from "fishery"
+import { Shape } from "../../src/schedule"
+
+export default Factory.define<Shape>(({ sequence }) => ({
+  id: `shape${sequence}`,
+  points: [{ shape_id: `shape${sequence}`, sequence: 1, lat: 0, lon: 0 }],
+  stops: [{ id: "1", name: "Some Stop", lat: 0, lon: 0 }],
+}))
```

Comment:
question: can the stop factory be used here?

Reply:
also here https://github.com/mbta/skate/blob/5b0d06f69d8b742aabfcc9de98a4e7ca68ea7934/assets/tests/factories/shape_data.ts#L7

Reply:
Addressed in 234074e8.